include/srdfdom/model.h

```diff
@@ -265,7 +271,8 @@ class Model
   std::vector<VirtualJoint> virtual_joints_;
   std::vector<EndEffector> end_effectors_;
   std::vector<LinkSpheres> link_sphere_approximations_;
-  std::vector<DisabledCollision> disabled_collisions_;
+  std::vector<std::string> no_default_collision_links_;
+  std::vector<CollisionPair> collision_pairs_;
```

Comment:
```suggestion
  std::vector<CollisionPair> disabled_collision_pairs_;
  std::vector<CollisionPair> enabled_collision_pairs_;
```
I believe this would make the code below somewhat nicer and you could drop the `disabled_` flag in each object.

Feel free to disregard this comment if you disagree.

Reply:
If you split this information into two separate members, you need to duplicate the corresponding code to handle them, don't you? Where exactly do you see a simplification?

Reply:
I guess I mostly dislike [passing a bool to a function just to filter a list](https://github.com/ros-planning/srdfdom/pull/97/files#diff-e7f7ff52129fe82e20745eef9247c07c322e01e63aa6debc6edad556b7876ca3R353). You might just as well pass in the separate lists as arguments and avoid iterating over data you know you'll skip...

Again, I don't feel strongly about it.

Reply:
I see your point. I have split the vectors now.